test/cases.yml

```diff
@@ -10,13 +10,19 @@
   steps:
     -   in: bn deploy -p /home/dockeruser/test/trollycopter icydiceymolecule
         err: "ENOENT: no such file or directory, open '/home/dockeruser/test/trollycopter/binaris.yml'"
+        out: |-
+            
+            Tip: You can export BINARIS_LOG_LEVEL=[silly|debug|verbose] to view debug logs
```

Comment:
Consider not repeating this each time, and just putting a `*` in most of the outputs.